src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -31,6 +31,24 @@
         </context>
     </template>
 
+    <template name="pdv" description="debug println! value" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$EXPRESSION_COPY$ = {:?}&quot;, $EXPRESSION$);">
```

Comment:
Hmm, I believe some community feedback will be very useful here. Is this already in nightly? If so maybe I'll ask guys on reddit about this. What do you think?